### PR TITLE
Fixing warnings

### PR DESCRIPTION
**Pull request by piogor**
See, #308

### CODE REVIEW

**Comment by piogor** [comment]
@t-b , now the most of the buidl-time warnings are fixed. The only left are related to bibliography/citations:
```
checking consistency... /home/travis/build/tango-controls/tango-doc/source/reference/bibliography.rst:14: WARNING: Citation [TangoWeb] is not referenced.

/home/travis/build/tango-controls/tango-doc/source/reference/bibliography.rst:16: WARNING: Citation [ALBA] is not referenced.

/home/travis/build/tango-controls/tango-doc/source/reference/bibliography.rst:18: WARNING: Citation [Soleil] is not referenced.

...


/home/travis/build/tango-controls/tango-doc/source/reference/bibliography.rst:49: WARNING: Citation [Elettra] is not referenced.

/home/travis/build/tango-controls/tango-doc/source/reference/bibliography.rst:53: WARNING: Citation [ATKtutorial] is not referenced.
```
 which I am not sure what to do about. @andygotz, do we realy need them? These are left-overs from the old manual and are pointing to old or not existing stuff.

**Comment by piogor** [comment]
@t-b, I will be glad if you cross-check the PR before I merge it. However, if you do not find time by Monday (03.02), I will ask @mliszcz, to do it. (It is quite a number of commits, but I decided to do it as one PR instead of 20 separate as most of them are just kind of typo correction (except removing the theme sources).

File: source/conf.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
 import sys
 import os
 
+import sys; sys.setrecursionlimit(1500)
```
Comment (mliszcz) [inline source/conf.py]: It would be good to have a comment here (or in commit msg) that explains why we need such a high recursion limit.

Comment (piogor) [reply source/conf.py]: Thanks @mliszcz! It was needed while rearranging ToCs (to fix 'not included in any toc' warning). However, the final arrangement does not require it anymore. I will commnet-out the line and do merge.

Comment (piogor) [reply source/conf.py]:
```suggestion
# import sys; sys.setrecursionlimit(1500)
```

**Pull request merged**